### PR TITLE
CNCF link fix

### PR DESCRIPTION
**Pull request by hugovalente-pm**
##### Summary

Fix the CNCF link due to changes on their website.

##### Additional Information

<details> <summary>For users: How does this change affect me?</summary>
  
</details>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,7 +120,7 @@ Check the [systemd-journal plugin of Netdata](https://github.com/netdata/netdata
   <br />
   Netdata actively supports and is a member of the Cloud Native Computing Foundation (CNCF)<br />
   &nbsp;<br/>
-  ...and due to your love :heart: and :star:, <b>Netdata is leading the Observability category</b> at the <a href="https://landscape.cncf.io/card-mode?category=observability-and-analysis&grouping=no&sort=stars">CNCF landscape</a>!
+  ...and due to your love :heart: and :star:, <b>Netdata is leading the Observability category</b> at the <a href="https://landscape.cncf.io/?item=observability-and-analysis--monitoring--netdata">CNCF landscape</a>!
```
Comment (ilyam8) [inline README.md:123]: It looks like they remove sort by GH start, at least I couldn't find how to do that - if so, I think we need to remove the "is leading the Observability category" part.

Comment (hugovalente-pm) [reply README.md:123]: yes, they removed it.
I discussed that with @Aliki92 and we are to keep the text as it is on the website, didn't actually notice it was different here. let me update that

Comment (hugovalente-pm) [reply README.md]: updated

**Review by ilyam8 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ilyam8) [reply README.md]: Thanks!

**Pull request merged**